javaparser-core-testing/src/test/resources/com/github/javaparser/EscapeSequences.java

```diff
@@ -0,0 +1,32 @@
+package com.github.javaparser;
+
+public class EscapeSequences {
```

Comment:
Perhaps rename this class/file to make it explicitly obvious that this is a manually-run test rather than a unit test file.

Reply:
Scratch this idea... I have come back with fresh eyes and noticed it's within the `resources` directory, so my comment is not applicable! :raised_eyebrow: :+1: